.github/workflows/copy-solution-scripts.yml

```diff
@@ -0,0 +1,63 @@
+name: Copy solution scripts to rrp-workshop-exercises
+# Copy instructor_notes/solutions/*sh to rrp-workshop-exercises repo
+
+# This workflow will run when there are changes to solution scripts in THIS repo
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "instructor_notes/solutions/*.sh"
+
+jobs:
+  file-script-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code from rrp-workshop-exercises
+        uses: actions/checkout@v4
+        with:
+          repository: AlexsLemonade/rrp-workshop-exercises
+          token: ${{ env.DOCS_BOT_GITHUB_TOKEN }}
```

Comment:
We don't have this token in this repo. We would normally get this using OP, but I'm realizing we don't have the OP tokens in this repo either... So there are actually a number of steps that would need to happen for this to work, and I'm wondering if the juice is worth the squeeze.

Reply:
oh right, I did prematurely delete the OP step, but yes your call about said juice as you'd probably be the one to take those steps! I don't think script updates are realistically going to happen often enough for this to be* super necessary, so we can always keep this open for another time.